tests/template/create_template_test.go

```diff
@@ -0,0 +1,192 @@
+package template_test
+
+import (
+	"os"
+	"strings"
+
+	"github.com/nimblehq/gin-templates/tests"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = BeforeEach(func() {
+	tests.ChangeDirectory(TemplateGeneratedPath + "/test-gin-templates")
+})
+
+var _ = Describe("Create template", func() {
+	Context("given current directory", func() {
+		It("contains project name", func() {
+			dir, err := os.Getwd()
+			if err != nil {
+				Fail("Failed to get current directory: " + err.Error())
+			}
+
+			isContainProjectName := strings.Contains(dir, "gin-templates/tests/template/test-gin-templates")
```

Comment:
`isContainProjectName` is not syntactically correct. How about:

```suggestion
		directoryContainProjectName := strings.Contains(dir, "gin-templates/tests/template/test-gin-templates")
```

This variable would need to be different for each test though and it could be tedious e.g. `fileContainProjectName`

Reply:
Fixed in c9fd9e3f0cde380ae461289ef9e30c16b77fd178